include/FootPrint.h

```diff
@@ -50,6 +48,8 @@ class FootPrint {
 
     const StepList& getSteps() const;
 
+    StepList& getSteps();
```

Comment:
An additional question, why do you need this? Do you need to modify the list of steps?

Reply:
Yes we update this the step positions and timing after step adaptation for re-planning the DCM trajectory, If I remember correctly.

Reply:
Ok I see!